doc/source/user_guide/io.rst

```diff
@@ -3200,6 +3200,13 @@ pandas supports writing Excel files to buffer-like objects such as ``StringIO``
 Excel writer engines
 ''''''''''''''''''''
 
+.. deprecated:: 1.2.0
+
+   The `xlwt <https://pypi.org/project/xlwt/>`__
+   engine, the only engine that supports writing
+   in an xls format, is no longer maintained and
+   will be removed in a future version of pandas.
```

Comment:
Nitpick: it's the package that is no longer maintained, and the engine that will be removed (now it reads like the code in pandas that will be removed is no longer maintained)

Reply:
(now, since it's written this way in many places in your PR (and the previous one), it might not be worth it to change ..)

Reply:
I appreciate the nitpicks; and as long as it's not holding things up happy to work on it to get it right. What do you think of:

> The `` `xlwt <https://pypi.org/project/xlwt/>`__`` package is no longer maintained and will not be available in a future version of pandas. This is the only engine currently in pandas that supports writing in an xls format.